src/TeamSpeak3QueryApi/Parameters/Parameter.cs

```diff
@@ -1,7 +1,7 @@
 using System;
 using System.Diagnostics;
 
-namespace TeamSpeak3QueryApi.Net
+namespace TeamSpeak3QueryApi.Net.Parameters
```

Comment:
The parameters are part of the query client and therefore belong to the upper namespace.